data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,19 @@
+/* exported countValues */
+
+function countValues(stack) {
+  const stackArray = Array.from(stack.print());
+  let counter = stackArray.length;
+  const numbersArray = [];
+
+  while (counter > 0) {
```

Comment:
We don't want to use `stack.print()` for any of these solutions. Instead, you can use the condition that `stack.peek()` is not undefined. Increment the counter variable in the loop.